src/pages/HomeNCPage.vue

```diff
@@ -0,0 +1,11 @@
+<script setup lang="ts">
+    import HomeNCDekstop from '@/components/HomeNC/HomeNCDekstop.vue'
+    import HomeNCMobile from '@/components/HomeNC/HomeNCMobile.vue'
+
+    
+</script>
+
+<template>
+  <HomeNCDekstop v-if="!$vuetify.display.xs" />
+  <HomeNCMobile v-else />
```

Comment:
Le breakpoint mobile peut se changer si la valeur défaut ne convient pas
et c'est Desktop 👀 
```suggestion
  <HomeNCMobile v-if="$vuetify.display.mobile" />
  <HomeNCDesktop v-else />
```